VinylRecordsRepository.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VinylRecordsManager.Models;
+using VinylRecordsManager.Data;
+using VinylRecordsManager.View;
+using System.Runtime.Remoting.Messaging;
+
+namespace VinylRecordsManager.Data
+{
+    public class VinylRecordsRepository
+    {
+        List<VinylRecords> masterList = new List<VinylRecords>();
+        VinylRecords vinyl = new VinylRecords();
+        VinylRecordsView userEnteredId = new VinylRecordsView();

```

Comment:
We don't want to connect the View and the Model directly this way. The Controller is the glue to connect these two.